### PR TITLE
Removes chemicals ability to spawn gas

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Removes from some chems (oxygen, nitrogen, carbondioxide, nitrousoxide, plasma and phlogiston) the ability to spawn gases

## Why It's Good For The Game
Atmos can never be balanced if these effects are around, especially when plumbing is a thing. 

## Changelog

:cl:
del: Removes from some chems (oxygen, nitrogen, carbondioxide, nitrousoxide, plasma and phlogiston) the ability to spawn gases
/:cl:

### CODE REVIEW

**Comment by CRITAWAKETS** [comment]
What kind of atmos system are you even planning that's broken apart by a convoluted method of spawning in basic gases?

**Comment by SmArtKar** [comment]
You understand that you basically made phlogiston completely useless? It's whole point was in plasma fires, not heated(it's much cooler than it would've been due to burning plasma) air

**Comment by TheBonded** [comment]
wait so the only way to get gasses now is mining them on lavaland or having them spawned in?

**Comment by thgvr** [comment]
Lame

**Comment by ArcaneMusic** [comment]
We already tried doing this years ago, we should probably finish the job now.

**Comment by Cheshify** [comment]
I'm all for changes that improve atmos, I don't know how this improves the atmospherics experience though

**Comment by necromanceranne** [comment]
Ghilk has got the right of it to be concerned, especially since splashing these chems results in instant gas floods of varying temperatures that atmos itself simply cannot replicate for speed, isolation and simplicity (cobby nerfed a particularly disgusting trick I used to use with superheated liquid plasma and supercooled liquid oxygen grenades to put atmospheric floods to shame)

I feel it would be best to figure out what phlog should do instead but I also think chem gas spawning is too abusable compared to atmos itself. Gas mining with chemistry, I will say, is maybe more trouble than it is worth.

**Comment by realthomas69** [comment]
boring!

**Comment by wesoda25** [comment]
Bro what the actual fuck is atmos balance. Atmos techs hold the power to hole up in their department for half the round and interact with hardly anyone and then unleash literal hell upon the rest of the station, but no chemistry being able to spawn gasses is the issue?? Clearly the playerbase don't mind whatever type of bullshit chemists are capable of considering the upvote downvote ratio on this. 😡 😠

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 1]
```diff
@@ -396,7 +396,8 @@
 		return
 	var/turf/open/T = get_turf(holder.my_atom)
 	if(istype(T))
-		T.atmos_spawn_air("plasma=[created_volume];TEMP=1000")
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm:399]: This is literally making phlog useless, come on. Find a replacement.

**Comment by Mothblocks** [comment]
> Atmos can never be balanced if these effects are around, especially when plumbing is a thing.

explain

**Comment by Ghilker** [comment]
The gist of it is, I want to regulate the gas production so that it can only come from cargo, the SM or lavaland/icebox.
Having infine plasma with chems Basically for free is a bad way to get this, I can't control how much you can get because you can expand infinitely with plumbing.
There is a plan to lower the atmos tanks moles amount and have a constant stream of gases from cargo and the SM to fuel atmos and I wanted to start with the "changes a lot for this project, but not much for everyone else"

**Comment by necromanceranne** [comment]
I believe currently plasma sheets also spawn plasma...if burned in some way that isn't a melee attack. I believe they're inheriting the plasma datum stuff that results in it exploding instead of spawning burning gas (which is a bug), but it can still do that if you use something like a laser instead.

**Comment by LemonInTheDark** [comment]
> The gist of it is, I want to regulate the gas production so that it can only come from cargo, the SM or lavaland/icebox. Having infine plasma with chems Basically for free is a bad way to get this, I can't control how much you can get because you can expand infinitely with plumbing. There is a plan to lower the atmos tanks moles amount and have a constant stream of gases from cargo and the SM to fuel atmos and I wanted to start with the "changes a lot for this project, but not much for everyone else"

Is this related to burning plasma and o2 for fuel? It seems a bit strange to limit gas production just for its own sake.
I'm also not a huge fan of cutting off chem as an infinite supply even separate from all that. If you're working on what I think you're working on I'm not very happy with it, but surely there's a better way then just sealing off chem totally

**Comment by Serxule** [comment]
> The gist of it is, I want to regulate the gas production so that it can only come from cargo, the SM or lavaland/icebox. Having infine plasma with chems Basically for free is a bad way to get this, I can't control how much you can get because you can expand infinitely with plumbing. There is a plan to lower the atmos tanks moles amount and have a constant stream of gases from cargo and the SM to fuel atmos and I wanted to start with the "changes a lot for this project, but not much for everyone else"

we already have infinite plasma in atmos...? tf you mean, also the second half of this doesnt even make sense, are you saying atmos will have to rely on the SM (how even?) and cargo? and what atmos tanks? the canisters or the o2/plasma ones (nerfing those just sounds stupid)

**Comment by moocowswag** [comment]
> The gist of it is, I want to regulate the gas production so that it can only come from cargo, the SM or lavaland/icebox. Having infine plasma with chems Basically for free is a bad way to get this, I can't control how much you can get because you can expand infinitely with plumbing. 

You can expand infinitely with siphoning gas from icebox and i`m pretty sure plumbing takes alot more power than gas pumps and filters...

**Comment by YakumoChen** [comment]
The only gas that really matters from chemistry to convert to a gas (its plasma) can't even be produced naturally from chemistry, it has to come from grinding bars (which need to be mined or otherwise produced).
Oxygen etc is much more easily produced but was already infinite anyway from a variety of other sources (ie, planet atmos)

**Comment by AdamElTablawy** [comment]
This breaks firebombs, incendiary grenades ordered from cargo, the chemical phlogistion, and a few other things.

**Comment by LemonInTheDark** [comment]
I'm not interested in this, I am unconvinced of the harm in chems producing gas, or even gas generation in general. I'm happy with the ideas I've seen of gas shipments and such, but I don't understand the use in heavily limiting say plasma supplies. 
Even if it is something to be constricted, it seems more sane to restrict chem production in some way, rather then barring off a link between the two systems.

**Pull request closed without merging**